package/gluon-ebtables-source-filter/files/lib/gluon/ebtables/100-source-chain

```diff
@@ -0,0 +1 @@
+chain('SOURCE', 'DROP')
```

Comment:
The chain name SOURCE is very inexpressive, it should in some way describe the packets that are passed to the chain. Maybe something like LOCAL_FORWARD.